src/lib/plugin.js

```diff
@@ -159,7 +160,7 @@ module.exports = class PluginVirtual extends EventEmitter2 {
     this._validator.validateOutgoingMessage(message)
     this._safeEmit('outgoing_request', message)
 
-    const response = await this._rpc.call('send_request', this._prefix, [message])
+    const response = await this._rpc.call('send_message', this._prefix, [message])
     this._validator.validateIncomingMessage(response)
```

Comment:
If your peer only supports the old `send_message` function it will return `true`, and this line will trigger an error